PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -40,8 +40,7 @@
         prefix = '"' + libdir
         plen = len(prefix)
         
-        # TODO: python3 probably will break here
-        for line in cachedata.splitlines():
+        for line in str(cachedata).splitlines():
```

Comment:
Right thanks for the pointer. I've changed this to use our compat.exec_command which properly decodes bytes to string on python 3.